package.json

```diff
@@ -10,6 +10,7 @@
     "grunt": "0.4.5",
     "grunt-angular-templates": "0.5.7",
     "grunt-available-tasks": "0.4.3",
+    "grunt-cli" : "~0.1.13",
```

Comment:
I will dig on it a bit more but yes I added this to get around the permission issue and I am using that version to actually execute the grunt build (hence to why unit tests are failing). Right now I am using a node cartridge on the openshift online version since I can't get a jenkins instance running on the itos version. I opened a support ticket. The mean stack cartridges seem to be having issues as well.